ui/components/app/modals/confirm-remove-account/confirm-remove-account.test.js

```diff
@@ -1,56 +1,76 @@
 import React from 'react';
 import configureMockStore from 'redux-mock-store';
 import { fireEvent } from '@testing-library/react';
-import { EthAccountType } from '@metamask/keyring-api';
+import { BtcAccountType } from '@metamask/keyring-api';
 import { renderWithProvider } from '../../../../../test/lib/render-helpers';
-import { ETH_EOA_METHODS } from '../../../../../shared/constants/eth-methods';
+import { createMockInternalAccount } from '../../../../../test/jest/mocks';
+import { addressSummary } from '../../../../helpers/utils/util';
+import { getMultichainAccountLink } from '../../../../helpers/utils/multichain/blockExplorer';
 import ConfirmRemoveAccount from '.';
 
+global.platform = { openTab: jest.fn(), closeCurrentWindow: jest.fn() };
+
+const mockAccount = createMockInternalAccount({
+  id: 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3',
+  name: 'Account 1',
+  address: '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
+});
+
+const mockNonEvmAccount = createMockInternalAccount({
+  id: 'e3a1c914-0bf3-41b3-b569-7c00185ad982',
+  name: 'Account 1',
+  address: 'bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq',
+  type: BtcAccountType.P2wpkh,
+});
+
+const mockEvmNetwork = {
+  nickname: 'network',
+  isEvmNetwork: true,
+  chainId: 'eip155:99',
+  network: {
+    type: 'rpc',
+    chainId: '0x99',
+    rpcUrl: 'https://rpc.network',
+    rpcPrefs: {
+      blockExplorerUrl: 'https://explorer.network',
+    },
+  },
+};
+
+const mockNonEvmNetwork = {
+  nickname: 'network',
+  isEvmNetwork: true,
+  chainId: 'bip122:000000000019d6689c085ae165831e93',
+  network: {
+    chainId: 'bip122:000000000019d6689c085ae165831e93',
```

Comment:
Nit: We could use `MultichainNetworks.BITCOIN` here, since it makes it easier to know which network we are using (like mainnet vs testnet)